src/evidently/features/non_letter_character_percentage_feature.py

```diff
@@ -7,8 +7,11 @@
 
 
 class NonLetterCharacterPercentage(GeneratedFeature):
+    column_name: str
+
     def __init__(self, column_name: str):
```

Comment:
I think `__init__` function can be removed.

Reply:
It allows us to use positional args instead of kw-only